taxi/views.py

```diff
@@ -87,3 +89,46 @@
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverForm
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+def update_license(request, pk):
+    driver = get_object_or_404(Driver, id=pk)
+    if request.method == "POST":
+        form = DriverLicenseUpdateForm(request.POST, instance=driver)
+        if form.is_valid():
+            form.save()
+            return redirect("taxi:driver-list")
+    else:
+        form = DriverLicenseUpdateForm(instance=driver)
+
+    return render(request, "taxi/update_license.html", {"form": form})
+
+
+def assign_driver(request, pk):
```

Comment:
Consider adding a `login_required` decorator to the `assign_driver` function to ensure that only authenticated users can access this view.